lib/cms/templates.ts

```diff
@@ -79,3 +79,46 @@ export async function createTemplate(
     filePath
   );
 }
+
+export const TEMPLATE_TYPES = {
```

Comment:
Where's this list from?

Reply:
Yep directly from https://github.com/HubSpot/cli-lib/blob/f7d547abbde458d4c394deab48b1ac0bb5376bef/lib/constants.js#L130 !